discuss feedback (#68)
OR13/diem-draft-charter: dispatch/charter.md

Reviewer: ```suggestion
4. Recommending that the new work is of sufficient scope to be AD-sponsored (this does not mean an AD is required to sponsor the work).
```

parralelism with how the other bullet items.

Reviewer: "(this does not mean an AD is required to sponsor the work)" is not specific for this. Moreover, you also have this warning right after the list: 

"Guidance given by the DISPATCH working group is not binding."

Reviewer: ```suggestion
5. Creating a mailing list for further discussion of the proposal. 
```

parralelism with how the other bullet items.

Reviewer: The last item may be misinterpreted as only protocol work is in scope. May be add "If proposed protocol work"

Author: I think this is fine as is. Even BCPs and Experimentals have interoperability concerns... and sometimes Informationals. Though Experimentals and Informationals are very, very rare for DISPATCH.

experience of the previously chartered DISPATCH and SECDISPATCH WGs.

With respect to the WIT area, DISPATCH is not to be used for new work
relating to transport protocols within the scope of an existing working
group. A list of those working groups is to be maintained on the DISPATCH working

> groups IETF Wiki page by the DISPATCH chairs in consultation with the WIT ADs.

Reviewer: ```suggestion
group's IETF Wiki page by the DISPATCH chairs in consultation with the WIT ADs.
```

Reviewer: May also insert a link to the wiki for convenience: https://wiki.ietf.org/en/group/dispatch

Other IETF areas and IETF-maintained technologies may have their own process

> for considering new work and are out-of-scope for DISPATCH. The chairs will maintain

Reviewer: ```suggestion
for considering new work and are out-of-scope for DISPATCH. The WG chairs will maintain
```

> a list on the DISPATCH wiki of the other working groups with a dispatch function

Reviewer: ```suggestion
a list on the DISPATCH wG Wiki of the other working groups with a dispatch function
```

> a list on the DISPATCH wG Wiki of the other working groups with a dispatch function

Reviewer: ```suggestion
a list on the DISPATCH WG Wiki of the other working groups with a dispatch function
```

nit

covering other IETF areas and IETF-maintained technologies.

The DISPATCH chairs will place requirements on presentations and
may reject presentation requests if not appropriate for the
DISPATCH working group. The chairs will maintain a list of presentation

> requirements on the DISPATCH wiki.

Reviewer: ```suggestion
requirements on the DISPATCH WG Wiki.
```

> Precedence will be given to documents which have evidence of interest in the

Reviewer: ```suggestion
Precedence to present will be given to proposals which have evidence of interest in the
```

> form of active drafts and list discussion. An Internet-Draft shall not be

Reviewer: ```suggestion
form of active Internet-Draftts and list discussion. An Internet-Draft shall not be
```

> form of active Internet-Draftts and list discussion. An Internet-Draft shall not be

Reviewer: ```suggestion
form of active Internet-Drafts and list discussion. An Internet-Draft shall not be
```

nit

dispatched multiple times unless a substantially revised proposal is put
forth, including compelling new reasons for accepting the work. 

> While DISPATCH has traditionally processed new work proposals during

Reviewer: I would move this para to be before the text starting with "The DISPATCH chairs will place requirements"
